frontend/src/features/public-form/mutations.ts

```diff
@@ -98,25 +98,16 @@
     },
   )
 
```

Comment:
```typescript
  type SubmitStorageModeClearFormWithVirusScanningType = (
    args: SubmitStorageFormWithVirusScanningArgs,
) => Promise<SubmissionResponseDto>
type SubmitMultirespondentFormType = (
    args: Omit<SubmitMultirespondentFormWithVirusScanningArgs, 'publicKey'>,
) => Promise<SubmissionResponseDto>
type UpdateMultirespondentSubmissionType = (
    args: Omit<SubmitMultirespondentFormWithVirusScanningArgs, 'publicKey'> & {
        submissionId?: string
    },
) => Promise<SubmissionResponseDto>
type SubmitClearFormWithVirusScanningMutationArgs =
    | SubmitStorageModeClearFormWithVirusScanningType
    | SubmitMultirespondentFormType
    | UpdateMultirespondentSubmissionType

const useSubmitClearFormWithVirusScanningMutation = (
    f: SubmitClearFormWithVirusScanningMutationArgs,
) => ...


```

Reply:
haha. see how many cases we're handling because of all our un-cleaned-up rollouts......